src/odemis/gui/cont/tabs.py

```diff
@@ -1602,6 +1611,419 @@ def get_display_priority(cls, main_data):
+        if val is None:
+            return
+        # Set the move gauge with the movement progress percentage
+        self.panel.gauge_move.Value = val * 100
+
+    def _toggle_switch_buttons(self, currently_pressed=None):
+        """
+        Toggle currently pressed button (if any) and untoggle rest of switch buttons
+        """
+        for button in self.position_btns.values():
+            button.SetValue(1) if button == currently_pressed else button.SetValue(0)
+
+    def _enable_movement_controls(self, cancelled=False):
+        """
+        Enable/disable chamber move controls (position and stage) based on current move
+        :param cancelled: (bool) if the move is cancelled
+        """
+        stage = self.tab_data_model.main.stage
+        # Get current movement (including unknown and on the path)
+        self.current_position = getCurrentPositionLabel(stage.position.value, stage)
+        self._enable_position_controls(self.current_position, cancelled)
+        # Enable stage advanced controls on milling
+        self._enable_advanced_controls(True) if self.current_position == MILLING else self._enable_advanced_controls(False)
+
+    def _enable_position_controls(self, current_position=None, cancelled=False):
+        """
+        Enable/disable switching position button based on current move
+        """
+        # The move button should turn green only if current move is known and not cancelled
+        if current_position in self.position_btns.keys() and not cancelled:
+            currently_pressed = self.position_btns[current_position]
+            currently_pressed.icon_on = img.getBitmap(self.btn_toggle_icons[currently_pressed][1])
+            currently_pressed.Refresh()
+        else:
+            self._toggle_switch_buttons(currently_pressed=None)
+        # Define which button to disable in respect to the current move
+        disable_buttons = {LOADING: MILLING, IMAGING: None, MILLING: COATING, COATING: MILLING, LOADING_PATH: MILLING}
+        for movement, button in self.position_btns.items():
+            if current_position == UNKNOWN:
+                # Only enable loading button when the move is unknown
+                button.Enable() if movement == LOADING else button.Disable()
+            elif movement == disable_buttons[current_position]:
+                button.Disable()
+            else:
+                button.Enable()
+
+    def _enable_advanced_controls(self, enable=True):
+        """
+        Enable/disable stage advanced controls
+        """
+        self.panel.ctrl_milling.Enable(enable)
+        self.panel.stage_align_slider_aligner.Enable(enable)
+        for button in self.btn_aligner_axes.keys():
+            button.Enable(enable)
+
+    def _pause_axis_connectors(self, pause=True):
+        """
+        Pause angle axis connectors (when leaving/entering milling mode)
+        :param pause: (bool) True to pause, False to resume
```

Comment:
Currently the axis connector is not paused out of milling, the intention is to give a reading of the angle while it's disabled, and it's enabled only in milling. So probably no reason to pause it?